README.md

```diff
@@ -877,31 +877,86 @@ interface FIDAttribution {
 ```ts
 interface INPAttribution {
   /**
-   * A selector identifying the element that the user interacted with for
-   * the event corresponding to INP. This element will be the `target` of the
-   * `event` dispatched.
+   * A selector identifying the element that the user first interacted with
+   * as part of the frame where the INP candidate interaction occurred.
+   * If `interactionTarget` is an empty string, that generally means the
+   * element was removed from the DOM as part of the interaction.
    */
-  eventTarget?: string;
+  interactionTarget: string;
```

Comment:
I think they were originally optional because INP could be zero if interactionCount was greater than 0 but no event entries were dispatched. However, once we started also observing `first-input` entries, I think they should always be set.